src/utils/time.ts

```diff
@@ -132,6 +171,7 @@ export function getHoursAgoInSeconds(hours: number) {
   return nowInSec - hours * 60 * 60;
 }
 
+// @debt this is a duplicate of getCurrentTimeInUTCSeconds
```

Comment:
[OOS] `@debt` this is a duplicate of `getCurrentTimeInUTCSeconds`

Reply:
Can you just remove it and use the other one instead?

Reply:
There is quite a lot of `@debt` notations in the code, we should start fixing the problems and removing them.

Reply:
I’m not using it for anything, it’s just something unrelated to the rest of my changes that noticed in my work. It should be fixed as a new PR focussed on cleaning it up.

Which also goes more generally for things labelled debt, they should be raised/captured in backlog issues/similar so we can actually get them fixed.

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/243